web/src/App.jsx

```
@@ -204,7 +205,25 @@ const ThreadItem = memo(({ thread, isActive, onSelect, showPreviews, onPin, onAr
         }
       }}
     >
-      <Avatar name={name} />
+      <div
+        role="button"
```

Biome (2.3.13)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)